core/src/main/scala/cats/syntax/traverseFilter.scala

```diff
@@ -2,3 +2,12 @@ package cats
 package syntax
 
 trait TraverseFilterSyntax extends TraverseFilter.ToTraverseFilterOps
+
+trait TraverseFilterSyntaxBinCompat0 {
```

Comment:
We also want this to go to `cats.syntax.all` and `cats.implicits` right?

Reply:
also can we add at least a doctest for it?

Reply:
It was in `cats.syntax.all` :) I added it to cats.implicits now, also made a doctest. Because there's been a release in the meantime, I added back the new bincompat trait. I'll push the changes once sbt gives me a green light.